taxonomies/categories.txt

```diff
@@ -28059,6 +28634,14 @@ it:Oli d'oliva dall'Italia
 nl:Olijfolie uit Italië
 country:en:Italy
 
+<en:Olive oils
```

Comment:
under "Olive oils" we should only have olive oils, not products that contain olive oil. That's important for things like computing Nutri-Score.

Reply:
D'accord, je le reclasse en en:Vegetable oils. Merci !